src/hmac/rtl/hmac_reg.rdl

```diff
@@ -157,10 +150,9 @@ addrmap hmac_reg {
 
     /* ---- HMAC384 Component Input LFSR Seed ---- */
     reg {
-        name = "HMAC384 component lfsr seed register type definition";
-        desc = "12 32-bit registers storing the 384-bit lfsr seed input.
-                These registers are located at HMAC384_base_address + 
-                0x0000_0130 to 0x0000_015C in big-endian representation.";
+        name = "HMAC384 LFSR SEED";
+        desc = "HMAC384 component lfsr seed register type definition
+                12 32-bit registers storing the 384-bit lfsr seed input.";
```

Comment:
big-endian

Reply:
no need for lfsr_seed since it's a random value